### ISSUE TITLE
Rendering Artifacts OpenglES2 

### ISSUE THREAD

**Issue by cejasmasmas** [memononen/nanovg#582]
Hi, first great library, thank you for it, I have an issue with artifacts while rendering,
doesn't happen on Windows the other platform i've tested, only on the raspberry pi3B, 
Everything works fine but, it would seem that it doesn't clear the stencil buffer or is misaligned
I took the color wheel from the example and I'm attaching an image, it looks like quads are drawing
behind whenever a "hole" is present
![dump](https://user-images.githubusercontent.com/65371443/81989459-7d040700-95fa-11ea-9c5b-fe19da42d102.png)

thank you

**Comment by mulle-kybernetik-tv** [comment]
When you are clearing the framebuffer, are you clearing all the relevant bits ? This works for me:

```
glClear( GL_COLOR_BUFFER_BIT|GL_DEPTH_BUFFER_BIT|GL_STENCIL_BUFFER_BIT);
```

**Comment by Calvin-L** [comment]
I recently had the same problem. In my case, the device had chosen a configuration with no stencil buffer. If possible, try adjusting your call to `eglChooseConfig`:

```c
const EGLint desiredAttributes[] = {
    ...
    EGL_STENCIL_SIZE, 8, // NanoVG requires the stencil buffer
    ...
    EGL_NONE
};
eglChooseConfig(..., desiredAttributes, ...);
```

**Comment by cejasmasmas** [comment]
thank you for your comments, the STENCIL BUFFER is ENABLED, thas is why I find this strange

**Comment by cejasmasmas** [comment]
So after a long time I found, a solution( I think.) I don't really get what the code is doing here other than, passing the stencil test
in order to draw the concave polygons. If someone can explain, I would appreciate it.

in this function
`static void glnvg__fill(GLNVGcontext* gl, GLNVGcall* call)`
in nanovg_gl.h, I changed 

`  glStencilOpSeparate(GL_FRONT, GL_KEEP, GL_KEEP, GL_INCR_WRAP);  `
` glStencilOpSeparate(GL_BACK, GL_KEEP, GL_KEEP, GL_DECR_WRAP);`

to

`glStencilOpSeparate(GL_FRONT, GL_KEEP, GL_KEEP, GL_INVERT);`
`glStencilOpSeparate(GL_BACK, GL_KEEP, GL_KEEP, GL_INVERT); `

now everything works even on the desktop/windows/linux ( it was already working there), I don't know if its going to break something else, at least until now everything is ok, I believe this can be closed now?
